<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>445</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>587</width>
    <height>445</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>587</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="main">
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>31</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>1</y>
        <width>651</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="zpP">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>45</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Расчет ЗП Программистам</string>
      </property>
     </widget>
     <widget class="QLabel" name="afterPay">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>241</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="beforePay">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>321</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ready">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>381</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Совершить выплату</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_p">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>151</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>141</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Ставка программистов</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_p"/>
         </item>
         <item>
          <widget class="QPushButton" name="ok1">
           <property name="text">
            <string>Принять</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Ставка дизайнеров</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_d"/>
         </item>
         <item>
          <widget class="QPushButton" name="ok2">
           <property name="text">
            <string>Принять</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>221</x>
        <y>11</y>
        <width>71</width>
        <height>361</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>221</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Текущий бюджет компании</string>
      </property>
     </widget>
     <widget class="QPushButton" name="zpD">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>78</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Расчет ЗП Дизайнерам</string>
      </property>
     </widget>
     <widget class="QPushButton" name="zpAll">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>111</y>
        <width>281</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Расчет ЗП Всем сотрудникам</string>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>203</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>51</y>
        <width>241</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Программисты</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Кол-во:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="cntP">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="stP">
                <property name="text">
                 <string>Ставка:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="stPP">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Редактировать</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Изменить ставку</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Дизайнеры</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Кол-во:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="cntD">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="stD">
                <property name="text">
                 <string>Ставка:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="stDD">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Редактировать список</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Изменить ставку</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>131</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>291</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Бюджет компании после выплат</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>-40</x>
        <y>361</y>
        <width>631</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>11</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>СОТРУДНИКИ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>РАСЧЁТ ЗП</string>
      </property>
     </widget>
     <widget class="Line" name="line_9">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>281</y>
        <width>351</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_8">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>411</y>
        <width>631</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="user">
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
     <widget class="QLabel" name="user_data">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>200</y>
        <width>601</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Данные</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="profile_image">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>140</width>
        <height>140</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>140</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>140</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>587</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Личный кабинет</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_p</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok2</sender>
   <signal>clicked()</signal>
   <receiver>label_p</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_d</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ok1</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_d</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok1</sender>
   <signal>clicked()</signal>
   <receiver>ok1</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok2</sender>
   <signal>clicked()</signal>
   <receiver>ok2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok1</sender>
   <signal>clicked()</signal>
   <receiver>label_p</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok2</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_3</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok2</sender>
   <signal>clicked()</signal>
   <receiver>label_8</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok1</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_p</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>ok2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok1</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_4</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_4</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_3</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok1</sender>
   <signal>clicked()</signal>
   <receiver>label_5</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>label_5</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>label_8</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>label_p</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>label_p</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
